src/NexusMods.App.UI/Controls/Trees/ModFileTreeViewModel.cs

```diff
@@ -205,12 +202,47 @@ public ModFileTreeViewModel(LoadoutId loadoutId, ModId modId, ILoadoutRegistry r
                     node => node.IsExpanded
                 ),
 
+                // File Count column
+                new TextColumn<IFileTreeNodeViewModel, string?>(
+                    Language.ModFileTreeViewModel_CreateTreeSource_File_Count,
+                    x => x.ToFormattedFileCount(),
+                    options: new TextColumnOptions<IFileTreeNodeViewModel>
+                    {
+                        // Compares IsFile, to show folders first, then by file count for folders, and file name for files.
```

Comment:
This comment is a leftover from above code which was pasted as reference

Reply:
What is the issue with the comment? I'm noting that we use FileCount for Folders and alphabetical for Files